### ISSUE TITLE
[Question] Documentation for non-plugin writers? (ie, the majority of use who use EDM)

### ISSUE THREAD

**Issue by ChrisMasterton** [googlesamples/unity-jar-resolver#538]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: 2019.3.19
  * External Dependency Manager version: 1.2.135
  * Source you installed EDM4U: tgz
  * Features in External Dependency Manager in use: Android Resolver, iOS Resolver, VersionHandler, etc.)
  * Plugins SDK in use: _____ (Firebase, Admob, Facebook, etc.)
  * Platform you are using the Unity editor on: _____ (Mac, Windows, or Linux)

### [REQUIRED] Please describe the question here:
Where is the documentation for the end-user of EDM (in plain English preferably)? Links to current documentation are written for those writing plugins that leverage EDM. But what about us dummies that are just trying to use the product because some other developer (hello ad networks) packaged their stuff with EDM?
It seems obvious to you when you wrote it, but when should I use Force Resolve over Resolve? Why should I use Version Handler? What on earth do all the options do and when should I use or care about them?

**Comment by paulinon** [comment]
Hi @ChrisMasterton,

Thank you for filing this ticket. It appears that our current documentation doesn't have clear answers to the questions you've listed, and I believe that adding more information would be helpful for a lot of developers. That being said, I'll be marking this as a feature request for now. You may refer to this thread for any updates.
